auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -26,24 +26,29 @@
       </div>
 
       <v-row no-gutters id="dashboard-actions" class="mb-n3">
-        <v-col cols="9">
+        <v-flex shrink class="mr-4">
```

Comment:
The v-flex made for much nicer dynamic page layout when the page size gets changed

Reply:
Don't use v-flex. [It's deprecated.](https://vuetifyjs.com/en/components/grids/#caveats)).

You can specify responsive  props for v-cols to make them look nice when the page size gets changed. [example](https://github.com/bcgov/business-create-ui/blob/7d29ed3ded310ac4a47e7de08ca53b9bacb69b6f/src/App.vue#L128)

Reply:
Done!